Tasks/Common/nuget-task-common/NuGetToolRunner2.ts

```diff
@@ -17,6 +17,7 @@ interface EnvironmentDictionary { [key: string]: string; }
 export interface NuGetEnvironmentSettings {
     credProviderFolder: string;
     extensionsDisabled: boolean;
+    quirks?: NuGetQuirks;
```

Comment:
This interface should only include details about how to prepare the environment. I'd prefer to change this to v2CredProviderPath or something like that and have `prepareNuGetExeEnvironment` just set the environment variables as needed.

Reply:
Makes sense - done